### ISSUE TITLE
Can not display tag details

### ISSUE THREAD

**Issue by niklas2810** [firefly-iii/firefly-iii#3839]
**Bug description**
I am running Firefly III version 5.4.1 (latest image, pulled 11am GMT), and my problem is:

This is the error message provided by Firefly III: `Argument 2 passed to FireflyIII\Support\Twig\AmountFormat::FireflyIII\Support\Twig\{closure}() must be of the type string, null given, called in /var/www/html/storage/framework/views/twig/02/02c17efe5f421b991c0a11b2e325ec2190dc08de3224bb170a4b173222ae7c79.php on line 193`. Path is `/tags/show/11`.

A complete stacktrace can be found [here](https://gist.github.com/niklas2810/6f5cb8e71e629c9c44a4f632ca32019d).

Debug information generated at 2020-09-23 18:05:10 Europe/Berlin for Firefly III version **5.4.1**.

| Scope | Version |
| --- | --- |
| Firefly III | 5.4.1 |
| Firefly III API | 1.4.0 |
| PHP | 7.4.10 |
| Host | Linux |

| System info | Value |
| --- | --- |
| Installation ID | 17ce7331-7b74-4692-87c3-58a81f1b8064 |
| Using docker? | true |
| Telemetry | true |
| Layout |  |
| App environment | local |
| App debug mode | false |
| App cache driver | file |
| App logging | notice, stack |
| Display errors | Off |
| Error reporting | ALL errors |
| Interface | apache2handler |
| Default language | en_US |
| Default locale | equal |
| BCscale | 12 |
| DB drivers | mysql, pgsql, sqlite |
| Current driver | pgsql |
| DB version | 15 (exp. 15) |
| Login provider | eloquent  |
| Trusted proxies (.env) | ** |

| User info | Value |
| --- | --- |
| Session start | 2020-09-01 00:00:00 |
| Session end | 2020-09-30 23:59:59 |
| Session first | 2020-05-15 00:00:00 |
| User ID | 1 |
| User language | de_DE |
| User locale | de_DE |
| Attempt at "de_DE.utf8" | 'de_DE.utf8' |
| Attempt at "de_DE.UTF-8" | 'de_DE.UTF-8' |
| User agent | Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:80.0) Gecko/20100101 Firefox/80.0 |

**Steps to reproduce**

Pull the latest version via Docker and run it.

Crashing tags: 1, 10, 11. All other tags (2-13) are being displayed as expected. Number 12+13, which do not have any transactions linked to them, work just fine. _When I link a transaction to Number 13, the instance crashes when visiting `/tags/show/13`. It works again after deleting the tag on the transaction._

**Bonus points**
- I searched and nobody reported this bug before
- I have added a stack trace from my log files
- I was able to replicate it on the demo site https://demo.firefly-iii.org/ (https://demo.firefly-iii.org/tags/show/39)

**Comment by niklas2810** [comment]
TL;DR: I guess this issue occurs for all tag indeces >= 10 which have transactions linked to them.

**Comment by JC5** [comment]
Nice find, fixed on the demo site, will be included in the next release :+1:
